test/rulesets/LinearAlgebra/norm.jl

```diff
@@ -99,6 +134,13 @@
         ȳ = rand_tangent(fnorm(x, p))
         @test extern(rrule(fnorm, zero(x), p)[2](ȳ)[2]) ≈ zero(x)
         @test rrule(fnorm, x, p)[2](Zero())[2] isa Zero
+        T == Float64 && sz == (3,) && @testset "Integer input, p=$p" begin
+            x = [1,2,3]
+            int_fwd, int_back = rrule(fnorm, x, p)
+            float_fwd, float_back = rrule(fnorm, float(x), p)
+            @test int_fwd ≈ float_fwd
+            @test unthunk(unthunk(int_back(1.0)[2])) ≈ unthunk(unthunk(float_back(1.0)[2]))
```

Comment:
nested thunks shouldn't occur in the output.
and i think they don't anymore

Reply:
Now removed.